configurations/za-staging.ts

```diff
@@ -20,7 +20,7 @@ const preEngagementConfig: PreEngagementConfig = {
     [
       {
         type: "InputItem",
-        label: "What is your name? (This may be just a screen name, or a nick name, if you are not comfortable giving us your real name)",
+        label: "What is your name? (This may be just a screen name, or a nick name, if you are not comfortable giving us your real name) /n We are here Monday – Friday, 11am-1pm & 2pm-6pm. If you need to speak to a Counsellor urgently, call our 24 hour Tollfree Number on 116.",
```

Comment:
@Humairaa127 were you intending this to be a backslash (\n)?  Do we want this to go out on the next deploy?

Reply:
Yes please include